packages/core/src/components/highlight/highlight.css

```diff
@@ -0,0 +1,63 @@
+.hds-highlight {
+  --accent-line-color: var(--color-bus);
+  --text-color: var(--color-black-90);
+  --width: 536px;
+  --margin: var(--spacing-m) 0;
+
+  border-left: 8px solid var(--accent-line-color);
+  color: var(--text-color);
+  display: flex;
+  justify-content: center;
+  align-items: space-between;
+  flex-direction: column;  
+  grid-row-gap: var(--spacing-m);
```

Comment:
Can a flex element have grid-row-gap? Chrome does not warn about it, but it does not seem to have any effect

Reply:
grid-row-gap adds space between rows: in quote between reference and text.